test/integration/install/spec.rb

```diff
@@ -0,0 +1,51 @@
+# Encoding: utf-8
+
+windows = os.windows?
+
+control 'basic_install' do
+  impact 0.4
+  title 'Install Splunk'
+  tag 'splunk_install'
+
+  describe package(os.windows? ? 'Splunk Enterprise' : 'splunk') do
```

Comment:
`windows` should be used.
Why I'm assigning `windows` instead of just using `os.windows?` isn't to save space, but because `os` isn't defined specifically when I use it here: `describe ... end unless os.windows?`
See: https://github.com/chef/inspec/issues/628